### PR TITLE
policy updates

### LINKED ISSUE #18461: Update privacy policies
Both are a bit outdated in terms of the used third-party providers and services.

### PR DESCRIPTION
**Pull request by jimaek**
- No more Google analytics
- Not sure why even had CDNs in the .com policy since that part is covered by the .net policy
- Added Gcore

fixes #18461

### CODE REVIEW

**Comment by MartinKolarik** [comment]
The website uses (will use) the cdn too, so it's relevant.

**Comment by jimaek** [comment]
>We use third-party Service Providers to serve all of our traffic under the domain cdn.jsdelivr.net.

We clearly define right now that there 2 policies and the user needs to select the domain. 

>The website uses (will use) the cdn too, so it's relevant.

Which CDN it will use? We can add that specific one to the .com policy

File: Privacy Policy - jsdelivr.com.md [hunk 1]
```diff
@@ -190,42 +190,6 @@ to provide the Service on our behalf, to perform Service-related services or to
 These third parties have access to your Personal Data only to perform these tasks on our behalf
 and are obligated not to disclose or use it for any other purpose.
 
-### Analytics
-
-We may use third-party Service Providers to monitor and analyze the use of our Service.
-
-#### Google Analytics
-
-Google Analytics is a web analytics service offered by Google that tracks and reports website traffic.
```
Comment (MartinKolarik) [inline Privacy Policy - jsdelivr.com.md:199]: This may need to be replaced by Plausible, or do we self host that?

Comment (jimaek) [reply Privacy Policy - jsdelivr.com.md:199]: Self host

**Comment by MartinKolarik** [comment]
Yes, the domain in the text was wrong, but I think we had both providers there to avoid the need to change the policy as we switch them.

> Which CDN it will use? We can add that specific one to the .com policy

I'm guessing Gcore, certainly not Bunny.

**Comment by MartinKolarik** [comment]
Merged with updated CDN providers for www to avoid having to update the text every time we switch them. Ping me in DM if you think we should keep just Gcore there.

**Pull request merged**